test/format/CONFIG.coverage

```diff
@@ -0,0 +1,7 @@
+# A reasonable configuration for coverage testing.
+btree.huffman_value=0
+cache.minimum=20
+runs.rows=1000000:5000000
+runs.threads=4:32
+runs.timer=15
+runs=100
```

Comment:
I'd remove `runs=100` (we don't want 100 runs, and it's being  overridden by the command line anyway).